app-misc/openrgb/files/OpenRGB-0.8-drop-bundled-libs.patch

```diff
@@ -0,0 +1,63 @@
+From f793240552b39ca5fe81b8b9e65242905ba012d1 Mon Sep 17 00:00:00 2001
+From: Marco Scardovi <mscardovi@icloud.com>
+Date: Sun, 18 Dec 2022 21:19:26 +0100
+Subject: [PATCH] Drop libs
+
+---
+ OpenRGB.pro                                                 | 3 +--
+ .../OpenRGBNanoleafScanningThread.cpp                       | 6 +++---
+ 2 files changed, 4 insertions(+), 5 deletions(-)
+
+diff --git a/OpenRGB.pro b/OpenRGB.pro
+index aabbd161..96aac7c6 100644
+--- a/OpenRGB.pro
++++ b/OpenRGB.pro
+@@ -63,11 +63,9 @@ INCLUDEPATH +=
+     dependencies/CRCpp/                                                                         \
+     dependencies/hueplusplus-1.0.0/include                                                      \
+     dependencies/hueplusplus-1.0.0/include/hueplusplus                                          \
+-    dependencies/httplib                                                                        \
+     dependencies/json/                                                                          \
+     dependencies/libe131/src/                                                                   \
+     dependencies/libcmmk/include/                                                               \
+-    dependencies/mdns                                                                           \
+     i2c_smbus/                                                                                  \
+     i2c_tools/                                                                                  \
+     net_port/                                                                                   \
+@@ -1562,6 +1560,7 @@ contains(QMAKE_PLATFORM, linux) {
+     -lmbedx509                                                                                  \
+     -lmbedtls                                                                                   \
+     -lmbedcrypto                                                                                \
++    -httplib
+ 
+     COMPILER_VERSION = $$system($$QMAKE_CXX " -dumpversion")
+     if (!versionAtLeast(COMPILER_VERSION, "9")) {
+diff --git a/qt/OpenRGBNanoleafSettingsPage/OpenRGBNanoleafScanningThread.cpp b/qt/OpenRGBNanoleafSettingsPage/OpenRGBNanoleafScanningThread.cpp
+index 1e3c926b..27565b78 100644
+--- a/qt/OpenRGBNanoleafSettingsPage/OpenRGBNanoleafScanningThread.cpp
++++ b/qt/OpenRGBNanoleafSettingsPage/OpenRGBNanoleafScanningThread.cpp
+@@ -10,7 +10,7 @@
+ #include <ifaddrs.h>
+ #endif
+ 
+-#include "mdns.h"
++#include <mdns.h>
```

Comment:
This shouldn't be necessary

Reply:
How do you call /usr/include/mdns.h without an #include?

Reply:
Not without. `#include "mdns.h"` works as well. The patch is not needed.

Reply:
I've tried and fails every single time.. It works only in that way

Reply:
When you use "" it checks on libdirs you have specified before while <> tell it to look for system's libraries placed inside /usr/include